File: 프로그래머스/0/181928. 이어 붙인 수/이어 붙인 수.js
function solution(num_list) {
    const oddNumber = num_list.filter(el => el % 2 === 0).join('');
    const evenNumber = num_list.filter(el => el % 2 === 1).join('');
    const result = Number(oddNumber) + Number(evenNumber);
    
    return result;
}

File: 프로그래머스/0/181928. 이어 붙인 수/이어 붙인 수.py
def solution(num_list):
    odd_sum, even_sum = "", ""
    
    for num in num_list:
        if num % 2 == 0:
            even_sum += str(num)
        else:
            odd_sum += str(num)
            
    return int(even_sum) + int(odd_sum)